tests/AbstractTestCase.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Illuminate\Tests;
+
+use Mockery;
+use Illuminate\Support\Carbon;
+use PHPUnit\Framework\TestCase;
+
+abstract class AbstractTestCase extends TestCase
+{
+    /**
+     * @var \Illuminate\Support\Carbon
+     */
+    protected $now;
+
+    protected function setUp()
+    {
+        Carbon::setTestNow($this->now = Carbon::now('UTC'));
```

Comment:
As we just extend `\PHPUnit\Framework\TestCase` this is useless, because `\PHPUnit\Framework\TestCase::setUp` and `\PHPUnit\Framework\TestCase::tearDown` actually do nothing.

It's actually more a good practice: when you decide to extend something else, if you're already calling `parent::` functions, you have fewer line of codes to change and you know that everything will just work fine